src/server/scripts/Northrend/zone_grizzly_hills.cpp

```diff
@@ -854,6 +855,258 @@ class spell_infected_worgen_bite : public SpellScriptLoader
+
+            DoCast(me, SPELL_DETONATE);
+            me->RemoveAllAuras();
+            me->SetVisible(false);
+            me->GetMotionMaster()->MoveTargetedHome();
+        }
+    };
+
+    CreatureAI* GetAI(Creature* creature) const override
+    {
+        return new npc_rocket_propelled_warheadAI(creature);
+    }
+};
+
+enum WarheadSpells
+{
+    SPELL_WARHEAD_Z_CHECK               = 61678,
+    SPELL_WARHEAD_SEEKING_LUMBERSHIP    = 49331,
+    SPELL_WARHEAD_FUSE                  = 49181
+};
+// 49107 - Vehicle: Warhead Fuse
+class spell_vehicle_warhead_fuse : public SpellScriptLoader
+{
+public:
+    spell_vehicle_warhead_fuse() : SpellScriptLoader("spell_vehicle_warhead_fuse") { }
+
+    class spell_vehicle_warhead_fuse_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_vehicle_warhead_fuse_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_WARHEAD_Z_CHECK) || !sSpellMgr->GetSpellInfo(SPELL_WARHEAD_SEEKING_LUMBERSHIP) || !sSpellMgr->GetSpellInfo(SPELL_WARHEAD_FUSE))
+                return false;
+            return true;
+        }
+
+        void HandleDummy(SpellEffIndex /*effIndex*/)
+        {
+            if (Unit* caster = GetCaster())
+            {
+                caster->CastSpell(caster, SPELL_WARHEAD_Z_CHECK, true);
+                caster->CastSpell(caster, SPELL_WARHEAD_SEEKING_LUMBERSHIP, true);
+                caster->CastSpell(caster, SPELL_WARHEAD_FUSE, true);
+            }
+        }
+
+        void Register() override
+        {
+            OnEffectHitTarget += SpellEffectFn(spell_vehicle_warhead_fuse_SpellScript::HandleDummy, EFFECT_1, SPELL_EFFECT_SCRIPT_EFFECT);
+        }
+    };
+
+    SpellScript* GetSpellScript() const override
+    {
+        return new spell_vehicle_warhead_fuse_SpellScript();
+    }
+};
+
+enum WarheadDenonate
+{
+    SPELL_PARACHUTE                     = 66154,
+    SPELL_TORPEDO_EXPLOSION             = 49290,
+    NPC_ALLIANCE_LUMBERBOAT_EXPLOSIONS  = 27689
+};
+// 49250 - Detonate
+class spell_warhead_detonate : public SpellScriptLoader
+{
+public:
+    spell_warhead_detonate() : SpellScriptLoader("spell_warhead_detonate") { }
+
+    class spell_warhead_detonate_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_warhead_detonate_SpellScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_PARACHUTE))
+                return false;
+            return true;
+        }
+
+        void HandleDummy(SpellEffIndex /*effIndex*/)
+        {
+            Unit* caster = GetCaster();
+            Player* player = GetHitPlayer();
+
+            if (!caster || !player)
+                return;
+
+            player->ExitVehicle();
+            player->KnockbackFrom(caster->GetPositionX(), caster->GetPositionY(), 3, 40);
+            caster->CastSpell(player, SPELL_PARACHUTE, true);
```

Comment:
why not? in the sniff, the parachute cast packet came right after the knockback one
the result was like this SS: http://prntscr.com/9dzhp3
The player falls slowly, but the parachute visual doesn't appear. 
But it must be something in the spell , because I tried other ways and he never shows